spec/factories/provider_relationships.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :provider_relationship do
+    cohort { Cohort.find_or_create_by!(start_year: 2021) }
```

Comment:
Could we have something dynamic like `Date.current.month > 8 ?  Date.current.year : Date.current.year + 1`? 
this is assuming the start of the next cohort is each September

Reply:
Follow the school_cohort initialiser so we can set it or just use the current